ai/mcp-server.md

```diff
@@ -0,0 +1,200 @@
+---
+title: Integrate Tiger Cloud with your AI Assistant
+excerpt: Manage your services and optimize your schema and queries using your AI Assistant
+products: [cloud, self_hosted]
+keywords: [ai, mcp, server, security]
+tags: [ai]
+---
+
+import RESTPrereqs from "versionContent/_partials/_prereqs-cloud-account-only.mdx";
+import CLIINSTALL from "versionContent/_partials/_devops-cli-install.mdx";
+import MCPCOMMANDS from "versionContent/_partials/_devops-mcp-commands.mdx";
+import MCPCOMMANDSCLI from "versionContent/_partials/_devops-mcp-commands-cli.mdx";
+import GLOBALFLAGS from "versionContent/_partials/_devops-cli-global-flags.mdx";
+
+
+# Integrate Tiger Cloud with your AI Assistant
+
+The $MCP_LONG provides access to your $CLOUD_LONG resources through Claude and other AI Assistants. $MCP_SHORT 
+mirrors the functionality of $CLI_LONG and is integrated directly into the $CLI_SHORT binary. You manage your
+$CLOUD_LONG resources including $VPCs, $SERVICE_SHORTs, read replicas, and related infrastructure using natural language 
+from your AI Assistant. As $MCP_SHORT is integrated with the $COMPANY documentation, ask any question and you 
```

Comment:
Worth pointing out that some of these things (VPCs, read replicas) are not technically supported by the MCP server as of right now. But we do plan to implement them in the near future, so maybe fine to leave as-is...? Not sure - your call. Just figured I'd mention it.